backend/danswer/chat/chat_utils.py

```diff
@@ -508,3 +511,28 @@ def extract_citations_from_stream(
             yield DanswerAnswerPiece(answer_piece="[" + curr_segment)
         else:
             yield DanswerAnswerPiece(answer_piece=curr_segment)
+
+
+_MAX_MISC_TOKENS = 2000
+
+
+def get_max_misc_tokens_for_chat() -> int:
```

Comment:
what's the calculation again?
4k max context, 3.5k input 0.5k output? 2k for system prompt + history + final message template (without document context) + 1.5k (~3 chunks) for document context?

Can we make a comment of it?